src/lib/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -38,6 +40,61 @@ public DependencyFinderEngineHelper(IDependencyFilter assemblyFilter, MetadataRe
             _currentAssemblyName = _reader.GetString(assemblyDefinition.Name);
         }
 
+        private IList<AssemblyReferenceInformation> ComputeAssemblyReferences(MetadataReader metadataReader)
+        {
+            List<AssemblyReferenceInformation> refs = new List<AssemblyReferenceInformation>();
```

Comment:
Maybe it's a VS code fix, but we usually use `var` when the types are obvious.
Same for the other variables below.

Reply:
I am not a fan of style, but I will change to match the project style!